explorer/explorerroutes.go

```diff
@@ -158,19 +163,64 @@ func (exp *explorerUI) Home(w http.ResponseWriter, r *http.Request) {
 	} else {
 		bestBlock = blocks[0]
 	}
+	// Get fiat conversions if available
+	homeInfo := exp.pageData.HomeInfo
+	var conversions *nextReductionConversions
+	xcBot := exp.xcBot
+	if xcBot != nil {
+		conversions = &nextReductionConversions{
+			PowSplit:        xcBot.Conversion(dcrutil.Amount(homeInfo.NBlockSubsidy.PoW).ToCoin()),
+			TreasurySplit:   xcBot.Conversion(dcrutil.Amount(homeInfo.NBlockSubsidy.Dev).ToCoin()),
+			TreasuryBalance: xcBot.Conversion(dcrutil.Amount(homeInfo.DevFund).ToCoin()),
+		}
+	}
 
-	// Safely retrieve the current inventory pointer.
-	inv := exp.MempoolInventory()
-
-	// Lock the shared inventory struct from change (e.g. in MempoolMonitor).
-	inv.RLock()
-	exp.pageData.RLock()
+	str, err := exp.templates.exec("home", struct {
```

Comment:
The `Home` method got messed up. It shouldn't have changed.  I think something went wrong since your initial diff because `NextReduction` is using `homeConversions` instead of `nextReductionConversions`, and the `Home` handler is changed a lot.